src/main/webapp/app/exercises/shared/example-submission/example-submission-import/example-submission-import.component.html

```diff
@@ -44,6 +48,9 @@ <h4 class="modal-title" jhiTranslate="artemisApp.exampleSubmission.importExample
                         <td>
                             <jhi-result [participation]="submission.participation!"></jhi-result>
                         </td>
+                        <td>
+                            {{ getSubmissionSize(submission) }}
```

Comment:
We try to minimize the amount of function calls in html. Can you make the submissionSize an attribute of the submissions?

Reply:
yeah, I'm aware of it. I tried to avoid that, but I could not figure out how to do that. I will give a try for your suggestion